test/unit/puppet_class_importer_test.rb

```diff
@@ -249,10 +249,10 @@ def setup
   end
 
   test "should detect correct environments for import" do
-    org_a = FactoryBot.build(:organization, :name => "OrgA")
-    loc_a = FactoryBot.build(:location, :name => "LocA")
-    org_b = FactoryBot.build(:organization, :name => "OrgB")
-    loc_b = FactoryBot.build(:location, :name => "LocB")
+    org_a = FactoryBot.create(:organization, :name => "OrgA")
+    loc_a = FactoryBot.create(:location, :name => "LocA")
+    org_b = FactoryBot.create(:organization, :name => "OrgB")
+    loc_b = FactoryBot.create(:location, :name => "LocB")
```

Comment:
otherwise the test fails with `Ancestry::AncestryException: No child ancestry for new record. Save record before performing tree operations.`